src/coreclr/binder/bindertracing.cpp

```diff
@@ -325,7 +327,10 @@ namespace BinderTracing
                     else
                     {
                         result = Result::Failure;
-                        errorMsg.Printf(W("Resolution failed with HRESULT (%08x)"), m_hr);
+
+                        SString errorMsgUtf8;
+                        errorMsgUtf8.Printf("Resolution failed with HRESULT (%08x)", m_hr);
+                        errorMsg.Set(errorMsgUtf8.GetUnicode());
```

Comment:
```suggestion
                        errorMsg.Printf("Resolution failed with HRESULT (%08x)", m_hr);
```
Anything wrong with doing the simple thing here?

Reply:
Nope. I'll move to that.